src/modules/dataCenter.js

```diff
@@ -11,5 +11,10 @@ const getMarketCapData = async () => {
   return response.data
 }
 
-export { getMarketCapData };
+const getCoinData = async () => {
+  let response = await axios.get('/api/currencies')
+  return response.data
```

Comment:
i would return only the array, so `response.data.currencies` if that is the only thing from the response that will be used in the component